src/model/utils.rs

```diff
@@ -215,7 +215,7 @@ fn try_resolve(
     }
 }
 
-#[cfg(all(feature = "unstable_discord_api", feature = "model"))]
+#[cfg(all(feature = "unstable_discord_api"))]
```

Comment:
```suggestion
#[cfg(feature = "unstable_discord_api")]
```

Reply:
Ah drat, I thought I'd properly removed all of them. My bad; will fix as soon as I can.